packages/screenshot/ios/Plugin/Screenshot.swift

```diff
@@ -1,8 +1,38 @@
 import Foundation
+import WebKit
 
 @objc public class Screenshot: NSObject {
-    @objc public func echo(_ value: String) -> String {
-        print(value)
-        return value
+    private let webView: WKWebView
+
+    public init(webView: WKWebView) {
+        self.webView = webView
+    }
```

Comment:
Please do the same here and pass the plugin instance, see https://github.com/capawesome-team/capacitor-plugins/blob/b5088c73cef1e2be5d915d1ed7a0649865a3d3ca/packages/live-update/ios/Plugin/LiveUpdate.swift#L24